tests/lint/clang_format.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# check lastest change, for squash merge into master
+./tests/lint/git-clang-format.sh HEAD~1
```

Comment:
Maybe we should use [git merge base](https://git-scm.com/docs/git-merge-base) so we can handle multiple commits

Reply:
yeah I don't want to change the actual content of the lint steps in this PR. but I agree with that idea.